### ISSUE TITLE
create schema doesn't work

### ISSUE THREAD

**Issue by upasana-mittal** [semi-technologies/weaviate-python-client#27]
I tried to create schema with

    client = weaviate.Client(WEAVIATE_URL)
    r = client.create_schema(body)

It returned `r` as `None` but later when i tried to get schema and add anything under defined class, it threw error saying

`weaviate.exceptions.UnexpectedStatusCodeException: Creating thing       {'error': [{'message': "invalid thing: class 'Articlev1' not present in schema"}]}`

**Comment by fefi42** [comment]
@upasana-mittal can you provide me with a schema to reproduce this?

**Comment by upasana-mittal** [comment]
> @upasana-mittal can you provide me with a schema to reproduce this?

    {
    "class": "Category",
    "description": "Category an article is a type off",
    "properties": [
      {
        "cardinality": "atMostOne",
        "dataType": [
          "text"
        ],
        "description": "category name",
        "name": "name"
      }
    ]
      }

**Comment by fefi42** [comment]
Hello @upasana-mittal thanks for your answer.

With your schema I can create classes. 
``` python
import weaviate

schema = {
    "things": {
        "type": "thing",
        "name": "gh-27",
        "maintainer": "hello@semi.technology",
        "classes": [
        {
            "class": "Category",
            "description": "Category an article is a type off",
            "properties": [{
                "cardinality": "atMostOne",
                "dataType": ["text"],
                "description": "category name",
                "name": "name"
              }]
        }]
    }
}

client = weaviate.Client("http://localhost:8080")
client.create_schema(schema)
client.create_thing({"name": "Fruit"}, "Category")
```

I saw that your original class name was `Articlev1`. Class names must be understood by the contextionary and thus can not contain numbers. The function `client.create_schema` should actually throw an error if you try to create such a schema. If that is not the case please give me an example that I can reproduce.

**Comment by upasana-mittal** [comment]
> > @upasana-mittal can you provide me with a schema to reproduce this?
> 
> ```
> {
> "class": "Category",
> "description": "Category an article is a type off",
> "properties": [
>   {
>     "cardinality": "atMostOne",
>     "dataType": [
>       "text"
>     ],
>     "description": "category name",
>     "name": "name"
>   }
> ]
>   }
> ```

I was trying to post this json directly not as you created the structure for schema. And this didnt give me any error also. It returned `None`

**Comment by fefi42** [comment]
OK thank you. Yes that is indeed currently not checked. We will come up with a solution for this. We also saw that the documentation is not clear about how the schema must be defined. We will also change that.

**Comment by fefi42** [comment]
I just published a new version `0.3.5` that introduces schema validation. An incomplete schema will now throw a `SchemaValidationException`. 

I will close this for now feel free to reopen if something is not working as expected.
